magic/core/src/main/java/link/magic/android/modules/events/EventModule.kt

```diff
@@ -8,8 +8,53 @@ import org.web3j.protocol.core.Request
 import java.util.concurrent.CompletableFuture
 
 class EventModule(rpcProvider: RpcProvider) : BaseModule(rpcProvider) {
```

Comment:
Is it better to do enum here?

Reply:
I wanted to do that but then when it's used it would be `magic.events.eventsEnum.CLOSE_EVENT` for example, while now it is `magic.events.CLOSE_EVENT` which I think is better